src/olympia/api/paginator.py

```diff
@@ -49,3 +52,36 @@ def page(self, number):
                 raise EmptyPage('That page contains no results')
 
         return page
+
+
+class CustomPageNumberPagination(PageNumberPagination):
+    """Custom pagination implementation to hook in our `ESPaginator`."""
+    page_size_query_param = 'page_size'
+
+    def paginate_queryset(self, queryset, request, view=None):
```

Comment:
Bloody hell, I searched for this ages and I never saw this particular line. Thanks!